dev/io.openliberty.microprofile.telemetry.internal.common/src/io/openliberty/microprofile/telemetry/internal/common/cdi/OpenTelemetryProducer.java

```diff
@@ -51,12 +53,8 @@ private void init() {
     @ApplicationScoped
     @Produces
     public OpenTelemetryInfo getOpenTelemetryInfo() {
-        try {
-            OpenTelemetryInfoFactory factory = OSGIHelpers.getService(OpenTelemetryInfoFactory.class, OpenTelemetryProducer.class);
-            return factory.getOpenTelemetryInfo(metaData);
-        } catch (Exception e) {
-            return new ErrorOpenTelemetryInfo();
-        }
+        Optional<OpenTelemetryInfo> openTelemetryInfo = openTelemetryInfoFactoryService.call( (factory) -> {return factory.getOpenTelemetryInfo(metaData); });
+        return openTelemetryInfo.orElse(new ErrorOpenTelemetryInfo());
```

Comment:
This will always create an instance of `ErrorOpenTelemetryInfo`.  To be the same as before it should be:
```suggestion
        return openTelemetryInfo.orElseGet(ErrorOpenTelemetryInfo::new);
```